Src/Base/AMReX_FabArrayBase.cpp

```diff
@@ -718,7 +733,9 @@ FabArrayBase::FB::define_fb(const FabArrayBase& fa)
     {
 	const int   krcv = imap[i];
 	const Box& vbx   = ba[krcv];
+	const Box& vbx_ng  = amrex::grow(ba[krcv],1);
```

Comment:
We can change this to `const Box& vbx_ng = amrex::grow(vbx,1);`.

Reply:
Done